### ISSUE TITLE
Use Cmd + enter to publish a post 

### ISSUE THREAD

**Issue by ivanminutillo** [bonfire-networks/bonfire-app#397]
Should be quite easy with [hexdocs.pm/phoenix_live_view/bindings.html#key-events](https://hexdocs.pm/phoenix_live_view/bindings.html#key-events)

**Comment by mayel** [comment]
Should be quite easy with https://hexdocs.pm/phoenix_live_view/bindings.html#key-events

**Comment by ivanminutillo** [comment]
done
